setup.cfg

```
@@ -72,6 +71,8 @@ test =
   pytest-markdown
   pytest-mock
   pytest-plus
+tenacity =
```

maybe call something like
```suggestion
retry =
```

---

I was considering this but I think that we may want to have a basic retry implementation as default instead of noop. In that case having an extra named retry is misleading as it would be another type of retry.

Another issue that worries me a little bit is inability to disable/customize retry behavior. What if consumer of the library does not want any retries? In its current form they are out of options, in fact the behavior would change based on the presence of absence of tenacity, creating some divergence in behavior.

I think that ideally we should add a parameter to the Runtime() class constructor that allows customization of the retry behavior for all the code that may need it. 

Do you have an idea on how we can achieve this level of flexibility in a clean way?

---

Accept a decorator callable?